### DISCUSSION TITLE
Restart Interval

### DISCUSSION THREAD

**Discussion by lethalox** [0xERR0R/blocky#1015] [Ideas]
I am using blocky in docker.   Great tool!

I would be nice to set an environment variable to restart/pull the config file on a periodic basis.   That way when you are making changes  you don't have restart the container.

**Comment by shizunge** [comment]
I am not sure it fits your need. This lets you manage the lists/domains without restarting blocky.

See https://github.com/0xERR0R/blocky/discussions/1347
